tests/php/UtilGetLocalPathTest.php

```
@@ -0,0 +1,116 @@
+				}
+				if ( -1 !== $component ) {
+					return $parts[ $component ] ?? null;
+				}
+				return $parts;
+			}
+		);
+		Functions\when( 'wp_normalize_path' )->alias(
+			static function ( $path ) {
+				$path = str_replace( '\\', '/', $path );
+				$path = preg_replace( '|(?<=.)/+|', '/', $path );
+				if ( str_starts_with( $path, '//' ) ) {
```

PHPStan (2.2.7)

[warning] Parameter `#1` $haystack of function str_starts_with expects string, array<string>|string|null given.


(argument.type)